9.4/paper.tex

```diff
@@ -1083,14 +1083,14 @@ \section{How to cite \dealii}\label{sec:cite}
     \item cuSOLVER \cite{cusolver}
     \item cuSPARSE \cite{cusparse}
     \item Gmsh \cite{geuzaine2009gmsh}
-    \item GSL \cite{gsl2016}
-    \item Ginkgo \cite{ginkgo-web-page}
+    \item GSL \cite{Galassi2009,gsl-web-page}
+    \item Ginkgo \cite{Anzt2022,ginkgo-web-page}
     \item HDF5 \cite{hdf5}
     \item METIS \cite{karypis1998fast}
-    \item MUMPS \cite{ADE00,MUMPS:1,MUMPS:2,mumps-web-page}
+    \item MUMPS \cite{ADE00,MUMPS:1,MUMPS:2,MUMPS:3,mumps-web-page}
     \item muparser \cite{muparser-web-page}
     \item OpenCASCADE \cite{opencascade-web-page}
-    \item p4est \cite{p4est}
+    \item p4est \cite{p4est,burstedde2020parallel}
```

Comment:
I'm not conceptually opposed to adding these references, but we're already at 4 pages of references on 18 pages total. Do we need to cite *all* of these?

Reply:
We can fix this in another PR, but yeah - we really don't need four citations for MUMPS. They recommend we use either the 2002 or 2019 citation in the manual license section:

    P. R. Amestoy, A. Buttari, J.-Y. L’Excellent and T. Mary,
    Performance and scalability of the block low-rank multifrontal
    factorization on multicore architectures,
    ACM Transactions on Mathematical Software,
    Vol 45, Issue 1, pp 2:1-2:26 (2019)

We can handle that in a follow-up but in general I'd prefer just one citation per dependency.

Reply:
I'm also in favor with one citation per dependency.

We often cite a publication related to a library, and then the web page as a separate entity. We can get merge these occurrences by adding a note with a link to the library, something like
```
note = {Library available online at \url{link/to/library}}
```
as we do currently with gsl:
https://github.com/dealii/release-papers/blob/9b205e4ae2409f2b7f4a0af326217420136cd727/9.4/paper.bib#L777-L782

I can take care of it in a follow-up PR.

---

For p4est: We have the new function `find_point_owner_rank` that uses a feature described in the most recent p4est paper. We already have it in the bibliography, so mentioning it here one more time doesn't hurt. We can get rid of the reference again for the next release paper.

Reply:
I think having 2 references (website + a paper) for a dependency is okay, especially if this is how to project wants to be cited. I agree that we should reduce MUMPS though.